dashboard/app/controllers/projects_controller.rb

```diff
@@ -41,6 +41,27 @@ def angular
     render template: "projects/projects", layout: nil
   end
 
+  def load
+    return if redirect_applab_under_13(@level)
+    if current_user
+      channel = StorageApps.new(storage_id_for_user).most_recent(params[:key])
+      if channel
+        redirect_to action: 'edit', channel_id: channel
+        return
+      end
+    end
+
+    create_new
+  end
+
+  def create_new
+    return if redirect_applab_under_13(@level)
+    redirect_to action: 'edit', channel_id: create_channel({
+      name: 'Untitled Project',
```

Comment:
Is this something we should have as a constant somewhere ('Untitled Project'). I think we use it in a few places. Unfortunately, I think that might span across dashboard vs. shared and ruby vs. JS, so might not be any good solution to reuse.